main.go

```diff
@@ -25,11 +25,25 @@ var (
 	// StartTime is the start time of the exporter represented as a UNIX epoch
 	StartTime = time.Now().Unix()
 )
+
 var (
 	filter      = flag.String("filter", "", "Filter the results of the request")
 	pagesize    = flag.Int64("max_projects", 10, "Maximum number of projects to include")
 	endpoint    = flag.String("endpoint", ":9402", "The endpoint of the HTTP server")
 	metricsPath = flag.String("path", "/metrics", "The path on which Prometheus metrics will be served")
+
+	DisableArtifactRegistryCollector = flag.Bool("collector.artifact_registry.disable", false, "Disables the metrics collector for the Artifact Registry")
```

Comment:
For consistency with the other flags (and with other exporters that I have), please `camelCase` these variable names